pkg/btf/btf_test.go

```diff
@@ -44,6 +46,25 @@ func setupfiles() func(*testing.T, string, ...string) {
 	}
 }
 
+func listBtfFiles() ([]string, error) {
+	kernelVersion, err := forceConfiguredKernelVersion()
+	if err != nil {
+		return nil, err
+	}
+
+	_, testFname, _, _ := runtime.Caller(0)
```

Comment:
could we just pass the name through argument?

Reply:
But how can we do this ? 
We are looking for the project root absolute path so idk how this can be improved.
```go
	_, testFname, _, _ := runtime.Caller(0)
	testdataPath := filepath.Join(filepath.Dir(testFname), "..", "..", "testdata")
```

Reply:
perhaps I don't follow the logic.. looks like you are trying to get to some place in the sources based on test file name, which seems strange to me
if we need to access some source location, I think we should rather use or add variable to defaults package?
in any case more details on what you're trying to do would be helpful, thanks

Reply:
To give more documentation for this one. I copy and paste it from here

https://github.com/cilium/tetragon/blob/f22de557663314e870d5288fd851755752428776/pkg/btf/validation_test.go#L82-L84

Though, there is a helper to do so, but cannot be imported because of another circular import issue.

https://github.com/cilium/tetragon/blob/f22de557663314e870d5288fd851755752428776/pkg/testutils/path.go#L12-L15